src/app/MessageDef/MessageDefHelper.cpp

```diff
@@ -94,5 +95,33 @@ void DecreaseDepth()
     gPrettyPrintingDepthLevel--;
 }
 #endif
+
+#if CHIP_CONFIG_IM_ENABLE_SCHEMA_CHECK
+CHIP_ERROR CheckInteractionModelRevision(int & aTagPresenceMask, uint8_t aTag, TLV::TLVReader & aReader)
+{
+    VerifyOrReturnError(!(aTagPresenceMask & 1 << aTag), CHIP_ERROR_INVALID_TLV_TAG);
+    aTagPresenceMask |= (1 << aTag);
+#if CHIP_DETAIL_LOGGING
+    {
+        uint8_t interactionModelRevision = 0;
```

Comment:
Should this value be checked against the local revision or it's just informative?

Reply:
it is just informative here, thanks